components/slidingPanel/slidingPanel.js

```diff
@@ -280,5 +288,13 @@ export default class SlidingPanel extends Component {
         </div>
       </div>
     );
+
+    return global
```

Comment:
If it is mandatory, then please remove the global attribute and make it used all the time.

Reply:
Again, I'd prefer to not maintain the _backwards_ compatibility and just use _global_ for all cases if the assumption is that all should use Global.